src/main.cpp

```diff
@@ -755,18 +738,24 @@ int main(int argc, char *argv[]) {
 
 
   cout << dev_mgr.get_avail_dev_info(deviceIndex).name.c_str() << endl;
-  cout << "OpenCL version: " << dev_mgr.get_avail_dev_info(deviceIndex).ocl_version.c_str() << endl;
-  cout << "Memory limit: " << dev_mgr.get_avail_dev_info(deviceIndex).max_mem << endl;
-  cout << "WG limit: " << dev_mgr.get_avail_dev_info(deviceIndex).wg_size << endl << endl;
+  if (benchmark_mode == false) {
+      cout << "OpenCL version: " << dev_mgr.get_avail_dev_info(deviceIndex).ocl_version.c_str() << endl;
+      cout << "Memory limit: " << dev_mgr.get_avail_dev_info(deviceIndex).max_mem << endl;
+      cout << "WG limit: " << dev_mgr.get_avail_dev_info(deviceIndex).wg_size << endl << endl;
+  }
   dev_mgr.init_device(deviceIndex);
 
   string kernel_url;
   if (h5_check_object(filename, "kernel_url") == true) {
-    h5_read_string(filename, "kernel_url", kernel_url);
-    cout << "Reading kernel from file: " << kernel_url << "... " << endl;
+      h5_read_string(filename, "kernel_url", kernel_url);
+      if (benchmark_mode == false) {
+          cout << "Reading kernel from file: " << kernel_url << "... " << endl;
+      }
   }
   else if (h5_check_object(filename, "kernel_source") == true) {
-    cout << "Reading kernel from HDF5 file... " << endl;
+      if (benchmark_mode == false) {
+          cout << "Reading kernel from HDF5 file... " << endl;
+      }
```

Comment:
Why is this info not printed in benchmark mode?

Reply:
As one might want to log the console output during a benchmark run, it could lead to unexpected disk startups when the buffer is flushed to disk. This causes strange power spikes. As this output is primarily useful for debugging, I disabled the output in benchmark mode. Do you think it is necessary to output this information in benchmark mode?